integration-tests/tx-rollup-l2-address.spec.ts

```diff
@@ -59,7 +59,7 @@ CONFIGS().forEach(({lib, setup, protocol}) => {
       const contract = await op.contract();
       const symbolReturn = await contract.storage()
       expect(JSON.stringify(symbolReturn)).toEqual(JSON.stringify(UnitValue))
-      const methodCall = await contract.methods.default('txr1di1E5BgY6QSPx61eE62P7KW93ckGwH2e7', 'tz4VHgLiRx5ZZjwU2QaybHc11EMJk3NcyvVc', '1').send();
+      const methodCall = await contract.methods.default(txRollupAddress, 'tz4VHgLiRx5ZZjwU2QaybHc11EMJk3NcyvVc', '1').send();
```

Comment:
small nitpick, but the variable name `methodCall` isn't very descriptive. Consider renaming it to something that reflects what the contract is doing. Like `ticketDeposit` or `deposit` or something similar